packages/zcli-apps/src/lib/buildAppJSON.ts

```diff
@@ -84,6 +107,16 @@ export const getAppPayloadFromManifest = (appManifest: Manifest, port: number, a
   }
 }
 
+const getSettingsArr = (appSettings: any) => {
+  const s: any[] = []
+  Object.keys(appSettings).forEach((settingName: string) => {
+    const setting: Record<string, any> = {}
+    setting[settingName.toString()] = appSettings[settingName.toString()]
```

Comment:
We see `settingName` as `string` type so I think we don't need `toString()` here.

Reply:
typescript was complaining and this had to be enforced here.